recipes/trio-aiohttp/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "trio-aiohttp" %}
+{% set version = "0.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 15b6a9c820a9bf8aa7d0ce691a94e4ebec86ecd6aaf38dc7691c4deec465eff8
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.5.0
+    - pip
+    - trio-asyncio
+    - aiohttp
```

Comment:
Could we drop everything not used to build here?

Reply:
The setup.py has `install_requires` with these, so I think that they are needed.

Reply:
So that makes sense for having them in `run`, but we usually only add things from `setup_requires` in `host`.

Reply:
Ahh right, I was getting confused with another package.

Reply:
No worries 🙂